src/src/beneficiaries/beneficiaries.controller.ts

```diff
@@ -71,6 +71,42 @@ export class BeneficiariesController {
 		}
 	}
 
+	@Post('remove-ag-duplications')
+	@UseGuards(new AuthGuard())
+	async removeAGDuplications(
```

Comment:
remove or deactivate?

Reply:
Changed API end-point to `/beneficiaries/deactivate-duplicate-beneficiaries`